CEDS/MSSQL/0015-View-SchoolYearsDim-Create.sql

```diff
@@ -0,0 +1,24 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+IF EXISTS (
+        SELECT 1
+        FROM INFORMATION_SCHEMA.VIEWS
+        WHERE TABLE_SCHEMA = 'analytics'
+            AND TABLE_NAME = 'ceds_SchoolYearsDim'
+        )
+BEGIN
+    DROP VIEW analytics.ceds_SchoolYearsDim;
+END;
+GO
+
+CREATE OR ALTER VIEW analytics.ceds_SchoolYearsDim
+AS (
+    SELECT
+        Id AS DimSchoolYearId,
```

Comment:
In AMT we use the `Key` suffix for the unique identifier, although the document uses `Id`, in the other views we have been using `Key`. 
I think, in order to have a more meaningful unique identifier, it's safe to use the edfi.SchoolYearType.SchoolYearDescription field. So I would do:
`SchoolYearDescription AS SchoolYearKey, `
instead of
`Id AS DimSchoolYearId,`

Reply:
Working on it.